src/TrueOrFalse/Tools/PersonalTopic.cs

```diff
@@ -2,17 +2,9 @@
 {
     public static Category GetPersonalCategory(User user, CategoryRepository categoryRepo)
     {
-        var wikiName = user.Name + "s Wiki";
-        var counter = 1;
-        while (categoryRepo.Exists(wikiName))
-    {
-            wikiName = user.Name + "s Wiki (" + counter + ")";
-            counter++;
-        }
-            
         return new Category
         {
-            Name = wikiName,
+            Name = user.Name + "s Wiki",
```

Comment:
This is an error. Only on creation should this pattern apply?


If this code is valid, it indicates a code duplication smell. The building-topic-name rule should not be duplicated.